src/containers/statistics/Statistics.js

```diff
@@ -1,11 +1,77 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Statisitcs = () => {
-  return (
-    <div>
-      <h3>Aggregated statistics</h3>
-    </div>
-  );
+/* eslint-disable no-unused-vars */
+const allEmployees = (data) => {
+  const allEmployees = [];
+  data.forEach((comp) => {
+    if (comp.employees !== undefined) {
+      return allEmployees.push(comp.employees);
+    }
+  });
+  return allEmployees;
 };
 
-export default Statisitcs;
+const employeesTotal = (allEmployees) => {
+  allEmployees.reduce(function(accumulator, currentValue) {
+    return accumulator + currentValue;
+  });
+};
+
+const averageEmployees = (employeesSum, allemployees) => {
+  const compEmployeesCounter = allemployees.length;
+  return Math.round(employeesSum / compEmployeesCounter);
+};
+
+const allFundings = (data) => {
+  const allFundings = [];
+  data.forEach((comp) => {
+    if(comp.funding !== undefined) {
+      return allFundings.push(comp.funding);
+    }
+  });
+  return allFundings;
+};
+
+const fundingsTotal = (allFundings) => {
+  allFundings.reduce(function(accumulator, currentValue) {
```

Comment:
employeesTotal and fundingstotal do basically the same thing, is there need of two different functions?